recipes/hypsoreader/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "hypsoreader" %}
+{% set version = "0.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DevAlvaroF/HYPSO-Reading/archive/v{{ version }}.tar.gz
+  sha256: b671969fa801ce4a7c196e92a3a202eff691c5a90d237d501bdcb8443276168c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+    - setuptools >=61.0
+    - setuptools_scm  >=3.4
+  run:
+    - python >=3.9
+    - numpy >=1.24
+    - gdal =3.6.2
+    - cartopy =0.21.1
+    - pandas
+    - matplotlib
+    - geopandas =0.12.2
+    - rasterio =1.3.4
+    - pyproj =3.4.1
+
+test:
+  imports:
+    - hypsoreader
+  commands:
+    - pip check
+    - pip list
+  requires:
+    - pip
+
+about:
+  license: GPL-2.0-only
```

```suggestion
  license: GPL-2.0-or-later
```

Ref: https://github.com/DevAlvaroF/HYPSO-Reading/blob/865159683f11ca5a034a243de30cd56b7b2420bc/pyproject.toml#L20

---

Awesome! This looks really good now. I feel like I know what it's going on :)